integration/simple-cache-client.test.ts

```diff
@@ -571,6 +576,221 @@ describe('lists', () => {
 });
 
 describe('Integration Tests for operations on sets datastructure', () => {
+  it('should succeed for addElements with byte arrays happy path', async () => {
+    const setName = v4();
+    const addResponse = await momento.setAddElements(
+      INTEGRATION_TEST_CACHE_NAME,
+      setName,
+      [LOL_BYTE_ARRAY, FOO_BYTE_ARRAY]
+    );
+    expect(addResponse).toBeInstanceOf(CacheSetAddElements.Success);
```

Comment:
I wonder if we could DRY up these tests by putting some of these really common operations into functions.

Something like this in the `describe` block.

```
const addToSet = function(setName: string, values: string[] | Uint8Array[], ttl: CollectionTtl?) {
  const addResponse = await momento.setAddElements(
    INTEGRATION_TEST_CACHE_NAME,
    setName,
    values,
    ttl
  );
  expect(addResponse).toBeInstanceOf(CacheSetAddElements.Success);
};
```

Then use it like so:

```suggestion
    const setName = v4();
    addToSet(setName, [LOL_BYTE_ARRAY, FOO_BYTE_ARRAY]);
```

Reply:
gonna timebox this: I see more compilation errors than I wanna see atm.

Reply:
I propose that we punt on this, unless you think this is a blocker.

Also won't this code have to change with your refactor PR anyway?

Reply:
I think the suggestion is a good one but I would be fine punting or fast-following it if we are happy with this PR otherwise.